albumentations/augmentations/functional.py

```diff
@@ -786,6 +786,17 @@ def gamma_transform(img, gamma):
     return img
 
 
+@preserve_shape
+def gamma_invert_transform(img, gamma):
+    if img.dtype == np.uint8:
+        table = (1 - ((1 - np.arange(0, 256.0 / 255, 1.0 / 255)) ** gamma)) * 255
+        img = cv2.LUT(img, np.round(table).astype(np.uint8))
+    else:
+        img = 1 - np.power(1 - img, gamma)
```

Comment:
I think better to take max value from `MAX_VALUES_BY_DTYPE` because img might have int32 dtype